s3_website.yml

```diff
@@ -0,0 +1,59 @@
+s3_id: <%= ENV['AWS_ACCESS_KEY'] %>
+s3_secret: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+s3_bucket: adb-mongolia-nua
```

Comment:
If you make this an environment variable, CI can push to separate buckets depending on if `master` or `develop`.